src/renderer/src/hooks/useOverlay.ts

```
@@ -0,0 +1,22 @@
+import { useState, useCallback, useMemo } from 'react'
+
+type Props = {
+  isOpen: boolean
+  onClose: () => void
+}
+
+const useOverlay = (render: (props: Props) => React.ReactNode) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)